pkg/utils/types.go

```diff
@@ -1,7 +1,11 @@
 package utils
 
 import (
+	"fmt"
+	"github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"
 	clientV1alpha1 "github.com/litmuschaos/chaos-operator/pkg/client/clientset/versioned"
+	volume "github.com/litmuschaos/kube-helper/kubernetes/volume/v1alpha1"
```

Comment:
Arrange the import path

Reply:
```
	"k8s.io/client-go/kubernetes"	
	"k8s.io/client-go/rest"
	corev1 "k8s.io/api/core/v1"
	clientV1alpha1 "github.com/litmuschaos/chaos-operator/pkg/client/clientset/versioned"
	volume "github.com/litmuschaos/kube-helper/kubernetes/volume/v1alpha1"
	"github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"

	"github.com/litmuschaos/chaos-executor/pkg/utils/k8s"
	"github.com/litmuschaos/chaos-executor/pkg/utils/litmus"
```

Reply:
I'm not able to get it, how are we arraging them?

Reply:
I think it is

Reply:
yes, A slight change it's like
- inbuilt lib
- k8s/otherpackage lib
- Own package import lib
```